pkgs/applications/office/envelope/default.nix

```diff
@@ -0,0 +1,61 @@
+{ stdenv
+, fetchFromGitHub
+, git
+, meson
+, ninja
+, pkgconfig
+, pantheon
+, python3
+, appstream-glib
+, desktop-file-utils
+, glib
+, gtk3
+, libgee
+, sqlite
+, wrapGAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "envelope";
+  version = "0.0.4";
+
+  src = fetchFromGitHub {
+    owner = "cjfloss";
+    repo = pname;
+    rev = version;
+    sha256 = "111lq1gijcm7qwpac09q11ymwiw2x3m12a28ki52f28fb1amvffc";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pantheon.vala
```

Comment:
this attribute is gone, as I've mentioned.

Reply:
> Can you add `doCheck = true;` so we can run the test?
> 
>     * https://github.com/cjfloss/envelope/blob/master/tests/meson.build#L33

Interestingly the test failed to validate the appdata file. I updated the PR with the failed test but I have no clue about what I should do to correct that ?